doc/sphinx/source/community/release_strategy.rst

```diff
@@ -186,49 +186,57 @@ Detailed timeline steps
 
 These are the detailed steps to take to make a release.
 
-1. Populate the milestone
+#. Populate the milestone
 
    - The core development team will make sure it adds issues that it intends to work on as early as possible.
    - Any contributor is welcome to add issues or pull requests that they intend to work on themselves to a milestone.
 
 
-2. ESMValCore feature freeze
+#. ESMValCore feature freeze
 
    - A release branch is created and branch protection rules are set up so only the release manager (i.e. the person in charge of the release branch) can push commits to that branch.
    - The creation of the release branch is announced to the ESMValTool development team along with the procedures to use the branch for testing and making last-minute changes (see next step)
 
 
-3. Some additional testing of ESMValCore
+#. Some additional testing of ESMValCore
 
    - Run all the recipes (optionally with a reduced amount of data) to check that they still work
    - If a bug is discovered that needs to be fixed before the release, a pull request can be made to the main branch to fix the bug. The person making the pull request can then ask the release manager to cherry-pick that commit into the release branch.
 
 
-4. ESMValCore release
+#. ESMValCore release
 
    - Make the release by following the :ref:`ESMValCore release instructions <esmvalcore:how-to-make-a-release>`.
    - Ask the user engagement team to announce the release to the user mailing list, the development team mailing list, on twitter
 
 
-5. ESMValTool feature freeze
+#. ESMValTool feature freeze
 
    - A release branch is created and branch protection rules are set up so only the release manager (i.e. the person in charge of the release branch) can push commits to that branch.
    - The creation of the release branch is announced to the ESMValTool development team along with the procedures to use the branch for testing and making last-minute changes (see next step)
 
 
-6. Some additional testing of ESMValTool
+#. Some additional testing of ESMValTool
 
-   - Run all the recipes to check that they still work and ask authors to review the plots
+   - :ref:`Run all the recipes to check that they still work and generate the overview HTML pages.<running_multiple_recipes>`
+   - Upload the results to the webpage at https://esmvaltool.dkrz.de/shared/esmvaltool/.
```

Comment:
Is having access to dkrz required to do that? Because if so I don't have access in there.

Reply:
Yes, that would be useful for you as our next release manager. I just sent you an email with information on how to create the account.

Reply:
I think I already got the permissions, as I was able to log in there. However, I am not sure how can all recipes be run in an automated way in Levante because it seems like cylc is not installed?

Reply:
On Mistral I was using the cylc module (outdated but working fine for recipe testing purposes). I also see that cylc is not part of the (very) limited number of modules available on Levante. I can try to request a cylc module on Levante at DKRZ, not sure how fast they would react. @bouweandela: shall I try or is there a way for a user to install cylc on their own?

Reply:
We can install cylc via conda-forge as well. There is a slight challenge in migrating from the old cylc 7 to the new version 8, but it should be doable. The migration guide is [here](https://cylc.github.io/cylc-doc/latest/html/7-to-8/index.html).

Reply:
Ok thanks! I'll try to install it myself and see how it goes. 

Is cylc installed in Jasmin though?

Reply:
I think the main problem with testing all recipes on Jasmin is that many recipes would fail because of missing Tier3 data.

Reply:
> But uploading the results to the webpage requires more permissions than just having access to DKRZ, right? I think only a handful of developers have access to the bot and the shared disk there. Not sure if and how we can open that a bit.

I think that the release managers need the access, and anyone helping them with testing.

Reply:
Should I add a note about passing on the accounts and permissions? Perhaps [here](https://docs.esmvaltool.org/en/latest/community/release_strategy.html#how-to-make-an-esmvaltool-release)?

Reply:
> Should I add a note about passing on the accounts and permissions? Perhaps [here](https://docs.esmvaltool.org/en/latest/community/release_strategy.html#how-to-make-an-esmvaltool-release)?

That could be useful as we regularly encounter this issue 👍